guildwars2/daily.py

```diff
@@ -27,6 +28,7 @@ class DailyMixin:
     async def daily_wvw(self, ctx):
         """Show today's WvW dailies"""
         embed = await self.daily_embed(["wvw"])
+        embed.set_thumbnail(url="https://render.guildwars2.com/file/2BBA251A24A2C1A0A305D561580449AF5B55F54F/338457.png")
```

Comment:
Try avoiding lines longer than 79, slice it a bit

Reply:
Reformatted with yapf